.env

```diff
@@ -1 +1,2 @@
 SASS_PATH="node_modules"
+REACT_APP_GITHUB_PERSONAL_ACCESS_TOKEN=7fb9bcb1891fab20e5ea8402dbefc9ef950dc4e3
```

Comment:
This should be in your `.env.local` file. It is best practice keep tokens or passwords private.